tests/test_circular_imports.py

```diff
@@ -51,7 +51,7 @@ def _discover_path_importables(
         if pkg_dir_path.parts[-1] == "__pycache__":
             continue
 
-        if all(Path(_).suffix != ".py" for _ in file_names):
+        if all(Path(_).suffix != ".py" for _ in file_names):  # pragma: no cover
```

Comment:
Make motivation obvious through code comments:

```suggestion
        if all(Path(_).suffix != ".py" for _ in file_names):  # It's never empty # pragma: no branch
```